tests/foreman/api/test_capsulecontent.py

```diff
@@ -1,7 +1,7 @@
 """Content Management related tests, which exercise katello with pulp
 interactions and use capsule.
 
-:Requirement: Capsule-Content
+:Requirement: ForemanProxy-Content
```

Comment:
```suggestion
:Requirement: Capsule-Content
```
also in other `:Requirement:`

Reply:
Resolved Now!